modules/gathering-kubelet-logs.adoc

```diff
@@ -0,0 +1,37 @@
+[id="gathering-kubelet-logs_{context}"]
+= Gathering kubelet logs
+
+.Procedure
+
+* Once the kubelet's log level verbosity is configured properly, you can gather logs by running:
```

Comment:
s/Once/After/

Reply:
s/by running:/by running the following commands:/

And if you end up using this suggestion, apply the same treatment to the remaining "run:" instances in the rest of the procedure (e.g., "run the following command:").